Gopkg.toml

```diff
@@ -0,0 +1,40 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/DataDog/datadog-go"
+  version = "1.1.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/armon/go-proxyproto"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/stripe/go-einhorn"
+
+[[constraint]]
+  name = "github.com/elazarl/goproxy"
+  source = "git@github.com:stripe/goproxy.git"
+  revision = "92f3b809c67db40d0cecefbf83bb76598614c8f3"
+  #revision = "0890fc15d1b6428a674b8252a2d238ca589daf73"
```

Comment:
Did you intend to leave this one commented out here? Adding a little more context about what that SHA is might be useful